pkg/controller/import-controller.go

```diff
@@ -803,6 +806,58 @@ func (r *ImportReconciler) getVddkImageName() (*string, error) {
 	return nil, errors.Errorf("found %s ConfigMap in namespace %s, but it does not contain a '%s' entry", common.VddkConfigMap, namespace, common.VddkConfigDataKey)
 }
 
+func (r *ImportReconciler) updatePVCBoundContion(pvc *corev1.PersistentVolumeClaim) {
+	// set bound condition by getting the latest event
+	events := &corev1.EventList{}
+
+	err := r.client.List(context.TODO(), events,
+		client.InNamespace(pvc.GetNamespace()),
+		client.MatchingFields{"involvedObject.name": pvc.GetName(),
+			"involvedObject.uid": string(pvc.GetUID())},
+	)
+
+	if err != nil || len(events.Items) == 0 {
```

Comment:
We should not eat the error here. I think it makes sense for this function to return an error. So something like:
```go
if err != nil {
  return err
}
```
You can sort and iterator over empty lists, so no need to check if events.Items is empty.